src/c4/ofpstream.hh

```diff
@@ -0,0 +1,86 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   c4/ofpstream.hh
+ * \author Mike Buksas
+ * \brief  Define class ofpstream
+ * \note   Copyright (C) 2016-2018 Los Alamos National Security, LLC.
+ *         All rights reserved.
+ */
+//---------------------------------------------------------------------------//
+
+//---------------------------------------------------------------------------//
+
+#ifndef c4_ofpstream_hh
+#define c4_ofpstream_hh
+
+#include <fstream>
+#include <vector>
+
+namespace rtt_c4 {
+
+//===========================================================================//
+/*!
+ * \class ofpstream
+ *
+ * Output stream for serializing output from all MPI ranks.
+ *
+ * This class is an output stream that serialized output from all MPI ranks.
+ * Output is inserted to the stream as with any other output stream. When the
+ * send() function is invoked, which *must be done synchronously across all
+ * processors*, all output inserted to the stream since the last call to send()
+ * is printed to the associated file, by MPI rank. In other words, all output
+ * inserted to an instance of ofpstream by rank 0 since the last call to send()
+ * is printed to the associated file, then all output inserted to the instance of
```

Comment:
This line is > 80 cols. Please fix. (Emacs: move mouse to this paragraph; M-q)

Reply:
Fixed by hand. M-q seems to reformat the comment to be longer than 80 in my current environment.